go.mod

```diff
@@ -24,52 +25,29 @@ require (
 	golang.org/x/sys v0.0.0-20191210023423-ac6580df4449 // indirect
 	golang.org/x/time v0.0.0-20191024005414-555d28b269f0
 	google.golang.org/grpc v1.24.0
-	gopkg.in/yaml.v2 v2.2.4
+	gopkg.in/yaml.v2 v2.2.8
 	helm.sh/helm/v3 v3.0.1
-	k8s.io/api v0.17.1
-	k8s.io/apiextensions-apiserver v0.17.1
-	k8s.io/apimachinery v0.17.1
-	k8s.io/apiserver v0.0.0
-	k8s.io/client-go v8.0.0+incompatible
-	k8s.io/code-generator v0.0.0
-	k8s.io/component-base v0.0.0
+	k8s.io/api v0.17.3
+	k8s.io/apiextensions-apiserver v0.17.3
+	k8s.io/apimachinery v0.17.3
+	k8s.io/apiserver v0.17.3
+	k8s.io/client-go v0.17.3
+	k8s.io/code-generator v0.17.3
+	k8s.io/component-base v0.17.3
 	k8s.io/klog v1.0.0
-	k8s.io/kube-aggregator v0.0.0
-	k8s.io/kube-openapi v0.0.0-20190918143330-0270cf2f1c1d
-	k8s.io/kubernetes v1.16.0
+	k8s.io/kube-aggregator v0.17.3
+	k8s.io/kube-openapi v0.0.0-20191107075043-30be4d16710a
 	sigs.k8s.io/controller-tools v0.2.4
 )
 
+// NOTE: k8s.io/Kubernetes v1.17 is currently vendored by hand in pkg/lib and should be
+// bumped along with other kube modules unitll dependency on k8s.io/Kubernetes is removed
+
 replace (
 	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309 // Required by Helm
 	github.com/openshift/api => github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
```

Comment:
Can this dependency be fixed as well?

Reply:
@ecordell https://github.com/operator-framework/operator-lifecycle-manager/pull/1370#discussion_r393301639